source/ways-of-working/standards/pull-requests.html.md.erb

```diff
@@ -130,6 +130,19 @@ All repositories in the organisation will automatically use this by default.
 
 You should start with this and adopt for your needs as needed by copying it into repository in the `.github` folder and editing it.
 
+### Suggested Branch Protection Rules
+
+These are the recommended settings to configure for your main branch on your repository, however each project might run differently so these are not These are the recommended settings to configure for your main branch on your repository, however each project might run differently so these are not compulsory.
+Check and fill out the below options on the Branch Protection Rules of your main branch.
+
+- Require a pull request before merging
+    - Require Approvals (minimal 1)
+    - Dismiss stale pull request approvals when new commits are pushed
+- Require status checks to pass before merging
+    - Require branches to be up to date before merging <br/>
+      Select `continuous-integration/jenkins/pr-merge` if using Jenkins or select relevant CI builds
+- Require conversation resolution before merging
```

Comment:
people can figure this out themselves, adding comments e.g. would be good to check this on later or references to why this was changes are good to do, but enforcing people mark resolve hides them from future viewers.

useful in some cases but I wouldn't make it a default
```suggestion
```